walletlibrary/src/test/java/com/microsoft/walletlibrary/did/sdk/credential/service/validators/OidcPresentationRequestValidatorTest.kt

```diff
@@ -106,7 +107,10 @@ class OidcPresentationRequestValidatorTest {
     @Test
     fun `valid signature is validated successfully`() {
         setUpExpiration(86400)
-        every { mockedPresentationRequest.getPresentationDefinition().credentialPresentationInputDescriptors } returns listOf(mockk())
+        every { mockedPresentationRequest.getPresentationDefinitions() } returns
```

Comment:
nit: When fixing old unit tests, can we add // Arrange, // Act, // Assert to align with new format?